docs/guides/compression_schemes.md

```diff
@@ -1,31 +1,22 @@
 # Compression Schemes
 
-## PTQ
-PTQ is performed to reduce the precision of quantizable weights (e.g., linear layers) to a lower bit-width. Supported formats are:
+## GPTQ Compression
 
-### [W4A16](../examples/quantization_w4a16/README.md)
-- Uses GPTQ to compress weights to 4 bits. Requires calibration dataset.
-- Optionally, [AWQ can also be leveraged for W4A16 quantization](../examples/awq/awq_one_shot.py)
-- Useful speed ups in low QPS regimes with more weight compression. 
-- Recommended for any GPUs types.
+Post-training quantization is performed to reduce the precision of quantizable weights (e.g., linear layers) to a lower bit-width. Supported formats are:
 
-### [W8A8-INT8](../examples/quantization_w8a8_int8/README.md)
-- Uses channel-wise quantization to compress weights to 8 bits using GPTQ, and uses dynamic per-token quantization to compress activations to 8 bits. Requires calibration dataset for weight quantization. Activation quantization is carried out during inference on vLLM.
-- Useful for speed ups in high QPS regimes or offline serving on vLLM. 
-- Recommended for NVIDIA GPUs with compute capability <8.9 (Ampere, Turing, Volta, Pascal, or older).
+| Scheme | Description | Data required? | vLLM Hardware Compatibility |
+|--------|-------------|----------------|-----------------------------|
+| **[W8A8-FP8](../examples/quantization_w8a8_fp8.md)** | 8-bit floating point (FP8) quantization for weights and activations, providing ~2X smaller weights with 8-bit arithmetic operations. Uses channel-wise quantization to compress weights to 8 bits, and uses dynamic per-token quantization to compress activations to 8 bits. Does not require calibration dataset. Activation quantization is carried out during inference on vLLM. Good for general performance and compression, especially for server and batch inference. | Does not require calibration dataset. | Latest NVIDIA GPUs (Ada Lovelace, Hopper, and later) and latest AMD GPUs. Recommended for NVIDIA GPUs with compute capability >=9.0 (Hopper and Blackwell) |
+| **[W8A8-INT8](../examples/quantization_w8a8_int8.md)** | 8-bit integer (INT8) quantization for weights and activations, providing ~2X smaller weights with 8-bit arithmetic operations. Uses channel-wise quantization to compress weights to 8 bits using GPTQ, and uses dynamic per-token quantization to compress activations to 8 bits. Good for general performance and compression, especially for server and batch inference. Activation quantization is carried out during inference on vLLM. Useful for speed ups in high QPS regimes or offline serving on vLLM. | Requires calibration dataset for weight quantization. | Supports all NVIDIA GPUs, AMD GPUs, TPUs, CPUs, and other accelerators. Recommended for NVIDIA GPUs with compute capability <8.9 (Ampere, Turing, Volta, Pascal, or older). |
```

Comment:
Same here, weight quantization can be both per tensor or per channel for int8.
Activations can also be static or not. 
Additionally, int8 activations can also be asymmetric. 

This does not change the hardware requirements but we should make note that these schemes are supported.